packages/eslint-plugin-query/src/rules/exhaustive-deps/exhaustive-deps.test.ts

```diff
@@ -298,6 +298,23 @@ ruleTester.run('exhaustive-deps', rule, {
         }
       `,
     },
+    {
+      name: 'should ignore references of the queryClient',
+      code: `
+        const CONST_VAL = 1
+        function useHook() {
+          const queryClient = useQueryClient()
+          const kueryKlient = useQueryClient()
+          useQuery({
+            queryKey: ["foo"],
+            queryFn: () => {
+                doSomething(queryClient)
+                doSomethingSus(kueryKlient)
```

Comment:
I just want to see if invoking a method on the queryClient is also fine:

```suggestion
                doSomething(queryClient)
                queryClient.invalidateQueries()
                doSomethingSus(kueryKlient)
```

Reply:
We already have a test case for that, but why not 🤷‍♂️